src/OpenTelemetry.Abstractions/Trace/ISpanScope.cs

```diff
@@ -0,0 +1,28 @@
+﻿// <copyright file="ISpanScope.cs" company="OpenTelemetry Authors">
+// Copyright 2018, OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// </copyright>
+
+namespace OpenTelemetry.Context
+{
+    using OpenTelemetry.Trace;
+
+    /// <summary>
+    /// Represents the current scope and its associated Span.
+    /// </summary>
+    public interface ISpanScope : IScope
```

Comment:
Why we need two layers? Is there a spec that says IScope must not have Span field?

Reply:
It limited the scope of the file changes. I can definitely add it directly to IScope but it will ripple into many other places. I also wasn’t sure if that IScope marker interface was intended for scopes other than spans. A bunch of existing tests would need patching etc.

If you want me to add it to IScope I will.

Reply:
I think we'll want this to just be on `IScope`, yeah. `Scope` basically acts as the local context.

Reply:
Ok, I can make that change..